content/blog/2021-01-28-survey-vaccines.Rmd

```diff
@@ -0,0 +1,354 @@
+it will also be critical to understand potential barriers to uptake or reasons
+for vaccine hesitancy. One major barrier may be concerns related to side
+effects, and we can track the percentages of those who may be concerned about
+side effects using one of our newly-added questions: *“How concerned are you
+that you would experience a side effect from a COVID-19 vaccination?”*
+
+When we examine the state-level estimated percentages (shown in the figure
+below), we find that the proportion of those who are very or somewhat worried
+about side effects vary from 30% to 50% in the U.S., and we observe the highest
+rates concentrated in the South.
+
+![](/blog/images/vaccine_blog_concern_sideeffects.svg)
+
+Based on our earlier results demonstrating differences in vaccination rates and
+acceptance by race and ethnicity, we also illustrate below how concerns about
+side effects compare between Black and White adults.
+
+![](/blog/images/vaccine_blog_concern_sideeffects_bw.svg)
+
+In all of the states with adequate survey sample sizes for both Black and White
+respondents, we find that Black individuals are consistently more likely to
+worry about experiencing a side effect compared to White individuals, as
+indicated by relative ratios that are greater than 1 (meaning a higher
+likelihood of worrying about a side effect). In fact, in some states, this ratio
+reaches 2, which means that Black adults are twice as likely to worry about
+experiencing a side effect compared to White adults When we further break down
+this comparison by vaccine acceptance/hesitancy (see the figure below), we find
+that Black individuals are always more likely to be concerned about experiencing
+side effects compared to White individuals, regardless of being
+vaccine-accepting or vaccine-hesitant. Future research is warranted to better
+understand how we can address these concerns in various subpopulation groups and
+if addressing concerns related to side effects can lead to higher vaccination
+rates.
+
+![](/blog/images/vaccine_blog_concern_sideeffects_hesitance.svg)
+
+## How You Can Analyze This Data
+
+In this post, we provided an introduction and an overview of the
+vaccination-related data that are available in the COVID Symptom Survey and how
+we can use the data to answer important public health questions related to
+vaccinations. There are, however, many more questions that you can help answer.
+For example, how do self-reported vaccinations and vaccination-related
+sentiments change over time? How do these change in response to guidelines and
+public health interventions? Is vaccine hesitancy associated with other
+behavioral measures or social activities? How is your local community responding
+to all of the vaccination-related efforts? You can answer all of these and more
+by analyzing these data!
+
+To get started, check out our [interactive
+map](https://delphi.cmu.edu/covidcast/?date=20210121&region=42003) and [survey
```

Comment:
we should remove the `?date=...` part. There is no need to fix those arguments

Reply:
done